tidb-cloud/ticloud-serverless-authorized-network-create.md

```diff
@@ -0,0 +1,50 @@
+---
+title: ticloud serverless authorized-network create
+summary: The reference of `ticloud serverless authorized-network create`.
+---
+
+# ticloud serverless authorized-network create
+
```

Comment:
Could we add an intro paragraph here to tell users what authorized network is?

Reply:
I think we need not. It is just a reference of CLI.
The introduction of the feature located here: https://docs.pingcap.com/tidbcloud/configure-serverless-firewall-rules-for-public-endpoints/#allow-aws-access